From 3b58a3fc1c1be2eef514f3ddbabd65628da6e57e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 00:17:46 +0800
Subject: [PATCH 7/7] fix(iris_bufmgr.c): set PAGE_SIZE as 16384 for LoongArch

Obviously not ideal, but this is simply meant as a preview/PoC.
---
 src/gallium/drivers/iris/iris_bufmgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gallium/drivers/iris/iris_bufmgr.c b/src/gallium/drivers/iris/iris_bufmgr.c
index 59261c96cff..e3879985553 100644
--- a/src/gallium/drivers/iris/iris_bufmgr.c
+++ b/src/gallium/drivers/iris/iris_bufmgr.c
@@ -95,7 +95,7 @@
  * included here.
  */
 #ifndef PAGE_SIZE
-#define PAGE_SIZE 4096
+#define PAGE_SIZE 16384
 #endif
 
 #define WARN_ONCE(cond, fmt...) do {                            \
-- 
2.43.4

